themes/default/content/what-is/what-are-docker-configs.md

```diff
@@ -0,0 +1,193 @@
+---
+title: What are Docker Configs?
+meta_desc: |
+     Learn more about what Docker Configs are and how to use them.
+
+type: what-is
+page_title: "What are Docker Configs"
+
+customer_logos:
+  title: Leading engineering organizations are building with Pulumi
+  logos:
+    - items:
+      - snowflake
+      - tableau
+      - atlassian
+      - fauna
+      - ware2go
+    - items:
+      - mindbody
+      - sourcegraph
+      - fenergo
+      - skai
+      - lemonade
+    - items:
+      - clearsale
+      - angellist
+      - webflow
+      - supabase
+      - ro
+---
+
+Docker, a leading platform in containerization technology, has revolutionized how applications are developed, shipped, and deployed. An essential facet of this ecosystem is the effective management of configuration data. [Docker Configs](https://docs.docker.com/engine/swarm/configs/) is a feature specially crafted to handle non-sensitive configuration information within Docker environments. This guide explores the ins and outs of Docker Secrets, highlighting its importance, functionality, and best practices.
+
+### What are Docker Configs?
+
+Docker Configs are a resource in Docker for storing non-sensitive information such as configuration files, separate from a service's image or running containers. This enables keeping Docker images as generic as possible without the need for bind-mounting configuration files into containers or using environment variables. Unlike Docker Secrets, Docker Configs are not encrypted at rest and are directly mounted into the container's filesystem.
+
+#### Key Features
+
+- **Separation of Configuration from Code**: Docker Configs allow you to store configuration files outside of your Docker images, leading to more generic and reusable images.
+- **Flexibility in Management**: Configs can be added, updated, or removed from services dynamically, without the need to rebuild or restart containers.
+- **Secure Transmission**: Configs are sent to the swarm manager over a mutual TLS connection and are stored securely.
+- **Easy Access within Containers**: Configs are automatically mounted into the container's filesystem, making them easily accessible to applications.
+- **Support for Various Data Types**: Configs can store generic strings or binary content, providing flexibility for different types of configuration data.
+
+### Creating Configs
+
+Docker Configs can be created via the Docker CLI.
```

Comment:
Is there another sentence that we want add here? In your previous Secrets one, you didn't use a h4 for install docker

Reply:
I will update this so it looks less weird.